src/backend/base/langflow/components/models/OpenAIModel.py

```diff
@@ -75,6 +76,13 @@ class OpenAIModelComponent(LCModelComponent):
             advanced=True,
             value=1,
         ),
+        HandleInput(
```

Comment:
Will add this to all the models if we like this direction

Reply:
I like this. 

Maybe we should create type alias to named OutputParser just lake the `LanguageModel`